packages/api/src/controllers/access-control.ts

```diff
@@ -36,6 +36,8 @@ const app = Router();
 type GateConfig = {
   refresh_interval: number;
   rate_limit: number;
+  viewer_limit_per_user: number;
```

Comment:
nit: WDYT of viewer_limit_for_user or user_viewer_limit? I was slightly confused with "per" for a second as it sounded like there would be multiple users in the stream (e.g. "playback users", even tho it wouldn't make sense).

Reply:
Updated